docs/maps/mountain.md

```diff
@@ -0,0 +1,42 @@
+<!-- replace _map_ with the actual map name -->
+<!-- change gamemode type for the Map data description  -->
+# Mountain
+
+***
+
+#### ![mountainicon](../assets/maps/mountain/mountain-icon.jpg)
+
+# Overview
+***
+- **Introduced:** v1.7.0
+- **Description:** A base that is split by a mountain.
```

Comment:
What does base refer to? The red and blue bases? or the center of the map? I think this could be reworded slightly